docs/concepts/policy/resource-quotas.md

```diff
@@ -46,8 +46,8 @@ Resource quota support is enabled by default for many Kubernetes distributions.
 enabled when the apiserver `--admission-control=` flag has `ResourceQuota` as
 one of its arguments.
 
-A resource quota is enforced in a particular namespace when there is a
-`ResourceQuota` in that namespace.
+Resource Quota is enforced in a particular namespace when there is at least one
+`ResourceQuota` object in that namespace.
```

Comment:
Whether this is default allow or default deny depends on the configuration ([`LimitedResources`](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/resourcequota/apis/resourcequota/types.go#L29))